assets/src/edit-story/components/canvas/useCanvasKeys.js

```diff
@@ -71,9 +72,15 @@ function useCanvasKeys(ref) {
     };
   }, [ref]);
 
-  useKeyDownEffect(ref, 'delete', () => deleteSelectedElements(), [
-    deleteSelectedElements,
-  ]);
+  useGlobalKeyDownEffect(
+    'delete',
+    () => {
+      if (!nativeEventHandlingExpected()) {
```

Comment:
This is already taken care of by the `useGlobalKeyDownEffect`, no? E.g. see [this code](https://github.com/google/web-stories-wp/blob/edfbf49c43014543e9f40dff9eb6aa9691cc3e5b/assets/src/edit-story/components/keyboard/index.js#L285). Are there some cases that are not handled there?

Reply:
Yes, for example when there is selection present in the document, that's not covered in the global key effect.